timely_beliefs/beliefs/utils.py

```diff
@@ -1054,22 +1054,57 @@ def meta_repr(
     )
 
 
+def convert_to_instantaneous(
+    df: "classes.BeliefsDataFrame",
+    boundary_policy: str,
+):
+    """Convert non-instantaneous events to instantaneous events.
+
+    Expects event_start as the sole index, and belief_time, source, cumulative_probability and event_value as columns.
+
+    :param df:              frame to convert
+    :param boundary_policy: 'min', 'max' or 'first'
+    """
+    df2 = df.copy()
+    df2.index = df2.index + df.event_resolution
```

Comment:
Could this be `df2.index - df.event_resolution`? This is related to the comment below.